config/Betterloadingscreen/tips/en_US.txt

```diff
@@ -1,66 +1,150 @@
+Press T while hovering on an item in NEI to see where it is in your world.
+Using a Circuit Assembler will make circuits cheaper!
+Using a Cleanroom will make circuits even cheaper!
+GT stands for Grind and Tedium
+Don't go out at night! Mobs can have up to 300 HP
+Steam is hot! It will burn Wooden Fluid Pipes
+Don't break rubber trees without a Treetap
+Make Muffler Moint Mup (Point up)
+Your EBF won't work if the muffler doesn't point to the sky!
+Eating too many of the same food will make you hate it!
+Abandon all vanilla knowledge all ye who enter here
+With the power of Duct Tape, I solved all of your maintenance problems!
+When you open a Twilight Portal, you'll get hit by lightning
+#Sometimes it's better to flee than fight some mobs
+Sometimes the best combat strategy is "Nigerundayooo!"
+You can power machines by putting a battery in their energy slots
+You can charge your jetpack by putting it in the Energy slot of LV machines
+Be prepared to take hours to craft a single machine!
+Oil + Thunder = Fire. Remember to put cobblestone over your oil wells!
+Nuclear Reactor configs give 10x the energy they give in IC2!
+Remember to plant cotton - you'll need a ton of it
+It's aluminIum, not aluminum
+#Sometimes it's better to buy than to craft
+Many ingots can only be smelted in the Electric Blast Furnace
+Use F7 to see spots where mobs can spawn!
+Use F9 to see chunk borders!
+Need seed oil for Forestry? Plant some peanuts
+Microcrafting is love, microcrafting is life
+Meta-Tip! You can see the list of loading screen tips in the config
+Hahahaha, coin quests go brrrrrrr
+If you're making a Project Red system, make sure to use a wall of drawers!
+How many circuits have you crafted in your life?
+Journeymap: press [J]
+Not by Vazkii
+sqrt(-1) loves you!
+(MultiMC > Curse) && (Technic > Curse): true
+All you had to do was tho follow the damn questline, CJ!
+Click on an item in the quest book to show its recipe in NEI
+Why not visit GregTech? We have... microcrafting
+Cables are | || || |_ y
+Varying foods gives hearts!
+You can change InGame Info XML scale in ingame configs
+Look for sulfur in the Nether
+Game mechanics datasheet: bit.ly/gtnh-datasheets
+If it's too tedious you're doing it wrong
+Do you also watch your oven while baking food?
+There's always something else you can do while waiting
+Use F3 + S to reload the sound if it's gone
+Life's all about opening issues, not closing them - Rongmario
+Batteries can be charged and discharged in the same slot
+3D-printed Lave 12 is the future - Sampsa
+Don't hold machines in your hotbar when playing with cables
+Was making HV connection. Missclicked and placed MV engraver - Lewissaber
+Destroyed once a Tier 1 rocket by accidentally pressing shift - Regian
+Visit r/feedthemems!
+I WANT ROCKET I WANT MOON - St4o
+Shift + Right Click on a GregTech Cover to open the menu
+Looking for Infused Stones? Try putting Amber in an Ore Finder Wand
```

Comment:
I suggest adding "Remove plates from machines using a crowbar"

Reply:
r/feedthemems -> r/feedthememes
Dropped the e